x-pack/plugins/fleet/server/services/epm/archive/parse.ts

```diff
@@ -164,18 +170,23 @@ export async function _generatePackageInfoFromPaths(
   paths: string[],
   topLevelDir: string
 ): Promise<ArchivePackage> {
-  const manifests: ManifestMap = {};
+  const manifestsAndRoutingRules: AssetsBufferMap = {};
   await Promise.all(
     paths.map(async (filePath) => {
-      if (filePath.endsWith(MANIFEST_NAME)) manifests[filePath] = await readFileAsync(filePath);
+      if (filePath.endsWith(MANIFEST_NAME)) {
```

Comment:
nit: the 2 conditions could be merged to one `filePath.endsWith(MANIFEST_NAME) || filePath.endsWith(DATASTREAM_ROUTING_RULES_NAME)`